api/README.md

```diff
@@ -0,0 +1,39 @@
+![alt tag](https://raw.github.com/JAVEO/knowler/master/docs/img/knowler.png)
+
+Knowler is the web tool to synchronize videos from presentations with slides. It is developed on akka-http and Polymer.
+
+1. Install and run mongoDb
+http://docs.mongodb.org/master/tutorial/install-mongodb-on-ubuntu/?_ga=1.90979482.1039490802.1437608880
+If you have problem 'ERROR: dbpath (/data/db) does not exist.' try this:
+http://stackoverflow.com/questions/24599119/mongodb-not-working-error-dbpath-data-db-does-not-exist
+
+2. Run mongo and create db called 'knowler':
+http://docs.mongodb.org/manual/tutorial/getting-started-with-the-mongo-shell/
+
+FLOW OF WORK WITH GITHUB:
+=========================
+(Assuming you have Github account)
```

Comment:
A to po co? Wszytko jest na githubie jak to wykonywać. Proponuję jeśli już tylko link dodać.